Fun.Build/StageContextExtensions.fs

```diff
@@ -499,6 +499,23 @@ module StageContextExtensions =
             | ValueSome(StageParent.Stage s) -> s.GetAllCmdArgs()
             | ValueNone -> []
 
+        /// <summary>
+        /// Return the remaining command arguments they are the arguments placed after <c>--</c> in the command line.
+        /// 
+        /// For example, if you run the following arguments:
+        /// 
+        /// <c> -p demo test1 v1 -- test2 v2</c> 
+        /// 
+        /// will return 
+        /// 
+        /// <c>[ "test2"; "v2" ]</c>
+        /// </summary>
+        member ctx.GetRemainingsCmdArgs() =
+            match ctx.ParentContext with
+            | ValueSome(StageParent.Pipeline p) -> p.RemainingCmdArgs
+            | ValueSome(StageParent.Stage s) -> s.GetRemainingsCmdArgs()
```

Comment:
Is the "s" after Remaining necessary? My English is not well to know that. But I see in the PipelineContext you are not using s for **RemainingCmdArgs**, so just wandering.

Reply:
Good catch.

I checked and Spectre.Console.Cli use `Remaining` for the name of property.

One less letter to type is always good 😊